src/pages/chainlink-nodes/v1/running-a-chainlink-node.md

```diff
@@ -66,13 +66,27 @@ Ganache is a mock testnet. Although you can run nodes on Ganache, it is not offi
 
 1. Create a local directory to hold the Chainlink data:
 
+   ```shell Sepolia
+   mkdir ~/.chainlink-sepolia
+   ```
+
```

Comment:
Can we present these commands for Sepolia and Goerli in tabs, since people will most likely do one or the other (not both)?

Reply:
Done

Reply:
It looks great!